<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>X2Power</class>
 <widget class="QDialog" name="X2Power">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>310</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>366</width>
    <height>310</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>366</width>
    <height>310</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AStarBox</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_12">
   <item>
    <widget class="X2Form" name="X2Form">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>99999</width>
       <height>99999</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>1</width>
       <height>1</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>300</width>
        <height>110</height>
       </rect>
      </property>
      <property name="title">
       <string>Circuit state when AStarBox boots</string>
      </property>
      <widget class="QCheckBox" name="checkBox_2">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>40</y>
         <width>64</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Port 2</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>40</y>
         <width>64</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Port 1</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_4">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>40</y>
         <width>64</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Port 4</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_3">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>40</y>
         <width>64</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Port 3</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_6">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>70</y>
         <width>130</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Dew Heater 2</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>120</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Dew Heater 1</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>8</y>
        <width>61</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Set</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>8</y>
        <width>120</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Dew heater 1 % :</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonCancel">
      <property name="geometry">
       <rect>
        <x>136</x>
        <y>240</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>40</y>
        <width>61</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Set</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>120</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Dew heater 2 % :</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonOK">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>232</x>
        <y>240</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="voltage">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>70</y>
        <width>250</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Input voltage : </string>
      </property>
     </widget>
     <widget class="QSpinBox" name="pwm5">
      <property name="geometry">
       <rect>
        <x>168</x>
        <y>8</y>
        <width>56</width>
        <height>24</height>
       </rect>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="pwm6">
      <property name="geometry">
       <rect>
        <x>168</x>
        <y>40</y>
        <width>56</width>
        <height>24</height>
       </rect>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>X2Form</class>
   <extends>QFrame</extends>
   <header>x2form.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
